build.gradle.kts

```diff
@@ -87,7 +87,7 @@ dependencies {
     implementation("com.fasterxml.jackson.module", "jackson-module-kotlin", "2.12.+")
 
     // Autohelp
-    implementation("me.schlaubi.autohelp", "kord", "1.2.0")
+    implementation("me.schlaubi.autohelp", "kord", "1.2.0-hotifx.1")
```

Comment:
Typo?

Reply:
Nope that version is called `1.2.0-hotfix.1`
https://schlaubi.jfrog.io/artifactory/forp/me/schlaubi/autohelp/kord/1.2.0-hotifx.1/